ray-on-gke/rayservice-examples/README.md

```diff
@@ -0,0 +1,263 @@
+# Serve meta llama2 (7b, 13b), llama2 70b quantized, falcon 7b or falcon40 quantized models
```

Comment:
Should we just put `rayservice-examples` under `ai-on-gke` directory so it's easier to find?

Reply:
Initially just wanted to group it with all the other ray related examples, not to clutter the root folder. I would be good with either